4.5/windows/windowsservercore/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM microsoft/windowsservercore
+
+ENV NPM_CONFIG_LOGLEVEL info
+ENV NODE_VERSION 4.5.0
+ENV NODE_SHA256 16aab15b29e79746d1bae708f6a5dbed8ef3c87426a9408f7261163d0cda0f56
+
+RUN powershell -Command \
```

Comment:
You're going to want to include `$ErrorActionPreference = 'Stop';` as the first command in these `powershell` blocks to ensure that failure stops the build appropriately. :+1:

(It's essentially POSIX's `set -e` for PowerShell.)